app/src/main/java/net/graphmasters/navigation/example/MainActivity.kt

```diff
@@ -198,13 +199,12 @@ class MainActivity : AppCompatActivity(), LocationListener,
                 "Cancel"
             ) { dialog, _ -> dialog.dismiss() }
             .show()
-
     }
 
     private fun initMapbox(savedInstanceState: Bundle?) {
         mapView?.onCreate(savedInstanceState)
         mapView?.getMapAsync { mapboxMap ->
-            mapboxMap.setStyle(Style.MAPBOX_STREETS) {
+            mapboxMap.setStyle("https://tiles.graphmasters.net/styles/nunav-light-mobile-v1.1/style.json") {
```

Comment:
But this in the build config, same as the API key. 
I've added this to a GitHub secret `MAP_STYLE_URL`, so you can read this in the build action [see here](https://github.com/Graphmasters/navigation-android-example/blob/75898a31b946a9465f46469a1f45762f9696c09e/.github/workflows/build.yml#L14-L14)